code/game/machinery/kitchen/microwave.dm

```diff
@@ -143,16 +143,16 @@
 	if(holdingitems && holdingitems.len >= limit)
 		usr << "The machine cannot hold anymore items."
 		return 1
-	else if(istype(O, /obj/item/weapon/storage/bag/plants))
+	else if(istype(O, /obj/item/weapon/storage/bag/plants) || istype(O, /obj/item/weapon/storage/bag/food/borg))
 		var/obj/item/weapon/storage/bag/B = O
-		for (var/obj/item/weapon/reagent_containers/food/snacks/grown/G in O.contents)
+		for (var/obj/item/weapon/reagent_containers/food/snacks/G in O.contents)
 			B.remove_from_storage(G,src)
 			if(contents && contents.len >= limit) //Sanity checking so the microwave doesn't overfill
 				user << "You fill the Microwave to the brim."
 				break
 
 		if(!O.contents.len)
-			user << "You empty the plant bag into the Microwave."
+			user << "You empty the [O] into the Microwave."

```

Comment:
\the formats for proper and improper nouns. So \the [x] is Rob, not the Rob, but the toolbox, not toolbox. Just the doesn't format.